account_payment_batch_oca/readme/USAGE.md

```diff
@@ -0,0 +1,11 @@
+You can create a Payment order via the menu Invoicing/Accounting \>
+Vendors \> Payment Orders and then select the move lines to pay.
+
+You can create a Debit order via the menu Invoicing/Accounting \>
+Customers \> Debit Orders and then select the move lines to debit.
+
+This module also adds an action *Add to Payment Order* on supplier
+invoices and *Add to Debit Order* on customer invoices.
```

Comment:
Does this Add to Payment Order button on vendor bill still exist? I don't see it.

Reply:
My bad, the button is visible as soon as a Payment Method is set on the invoice.